README.md

```diff
@@ -109,11 +115,21 @@ You can configure this template by passing different parameters in the Generator
 |securityScheme|Name of the security scheme. Only scheme with X509 and Kafka protocol is supported for now.|No|'mySchemeName'|
 |certFilesDir|Directory where application certificates are located. This parameter is needed when you use X509 security scheme and your cert files are not located in the root of your application.|No|`../not/in/my/app/dir`|
 
+## Setup Locally
```

Comment:
```suggestion
## Setup locally
```

lower case to be consisted with other sections

Reply:
Done ✔